src/providers/noop.rs

```diff
@@ -41,12 +39,12 @@ impl NoopProvider {
 }
 
 // Debug methods for testing
-#[cfg(test)]
-impl NoopProvider {
-    /// Get notifications for a specific token
-    pub fn get_notifications(&mut self, token: String) -> &Vec<String> {
-        self.bootstrap(token.clone());
-
-        self.notifications.get(&token).unwrap()
-    }
-}
+// #[cfg(test)]
```

Comment:
obvio no dead code - cool to fix later

Reply:
Yeah, I'll add it too the refactor issue